Sources/Types.swift

```diff
@@ -129,6 +156,24 @@ public class AnyProducer<A>: Producer {
 	}
 }
 
+public final class Fulfilled<Wrapped>: Producer {
```

Comment:
`Fulfilled` is really a term related to observables. In the context of a `Producer` abstraction, this should probably called `FixedProducer` or something like that.